### PR TITLE
Ci/actions cli

### PR DESCRIPTION
**Pull request by cosmic-vagabond**
* Fix CLI action to align with go 1.19

### CODE REVIEW

File: actions/cli/readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Ignite CLI Action
+
+This action makes the `ignite` CLI available as a Github Action.
```
Comment (jelysn) [inline actions/cli/readme.md:3]: can you update comment to be related to elys? (I think using `ignite` word would not be ideal here)

File: actions/cli/readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Quick start
+
+Add a new workflow to your repo:
```
Comment (jelysn) [inline actions/cli/readme.md:7]: `your repo` as well - `Elys`

File: actions/cli/readme.md (hunk 1 above)
Comment (cosmic-vagabond) [reply actions/cli/readme.md:3]: this is just a fork of the original folder from the ignite-cli repo

**Review by jelysn — APPROVED** [review]
LGTM

**Pull request merged**